ansible/roles/common/tasks/main.yml

```diff
@@ -63,3 +73,14 @@
     dest: /etc/nginx/nginx.conf
     regexp: 'server_names_hash_bucket_size'
     line: '	server_names_hash_bucket_size 64;'
+
+- name: nginx default site conf
+  template: src=nginx/default
+            dest=/etc/nginx/sites-enabled/default
+
+- name: remove default nginx site html page
```

Comment:
Showing the default nginx page is a bug in any and all circumstances, and I want that bug to read `404 not found`, not `hello, this is not configured properly`. Also, less unused files is better that more unused files.